src/main/java/convobot/commands/Add.java

```diff
@@ -0,0 +1,42 @@
+package commands;
+
+import tasks.Task;
```

Comment:
Perhaps can add a blank below to separate different packages? Same issues in several places too.

Reply:
Rejected. Adding a blank line causes checkstyle to fail.